spark/v2.4/spark/src/test/java/org/apache/iceberg/spark/source/TestPartitionValues.java

```diff
@@ -402,14 +402,13 @@ public void testPartitionedByNestedString() throws Exception {
 
     // input data frame
     StructField[] structFields = {
-        new StructField("struct",
-            DataTypes.createStructType(
-                new StructField[] {
-                    new StructField("string", DataTypes.StringType, false, Metadata.empty())
-                }
-            ),
-            false, Metadata.empty()
-        )
+        new StructField("col1",
+          DataTypes.createStructType(
+            new StructField[] {
+                new StructField("f1", DataTypes.StringType, false, Metadata.empty())
+            }
+                    ), false, Metadata.empty()
+            )
```

Comment:
Nearly all of the changes in this file are to formatting or names. Can you remove any changes that are not strictly necessary for what you're trying to fix?

Reply:
Yes. I will change.